attributes-info/rustc_macros/Cargo.toml

```diff
@@ -9,5 +9,5 @@ proc-macro = true
 [dependencies]
 synstructure = "0.12.1"
 syn = { version = "1", features = ["full"] }
-proc-macro2 = "1"
+proc-macro2 = "1.0.60"
```

Comment:
It doesn't seem like proper way to do it.
I think it's better just to update versions of `Cargo.lock` file. Just revert changes in `Cargo.toml`, call `cargo update` in `attributes-info` dir and commit changes in `Cargo.lock`